workshops/personal_website/README.md

```diff
@@ -214,33 +214,30 @@ Let's see what our website looks like so far:
 As you can see, the page is still blank. This is because we haven't added any
 **content** in the **`body`** section yet. Let's add some!
 
-### 5) Adding My Name to the Body of the HTML
+### 5) Adding a Name to the Body of the HTML
 
 Write your name in the **body** of the HTML page (between **`<body>`** and
-**`</body>`**). My name is `Drizzy Drake` so that's what I will write.
+**`</body>`**).
 
 > ![](img/html_add_name.gif)
 
-Notice that that the HTML code you write is immediately updated in the preview.
-That's why it's called a **"Live Preview"**
+Notice that that the HTML we write is immediately updated in the preview. That's
+why it's called a **"Live Preview"**
 
 ### 6) Adding a Description
 
-Underneath on a new line, write one short sentence about yourself. I love my job
```

Comment:
Why was this line removed? We should keep a text description of every step, since if we only have it in the GIF, it'll be inaccessible to people who can't view images, or are using a screen reader.